language/language_samples.rb

```diff
@@ -36,9 +38,11 @@ def sentiment_from_cloud_storage_file project_id:, storage_path:
 
   language  = Google::Cloud::Language.new project: project_id
   document  = language.document storage_path
```

Comment:
Add a newline after this line. I know it's not part of your change, can you remove extra spaces in front of `language` and `document`.